### ISSUE TITLE
Organize `test` folder

### ISSUE THREAD

**Issue by joeyhoer** [Compass/compass#1341]
The organization of the `test` folder leaves something to be desired.
https://github.com/chriseppstein/compass/tree/stable/test/fixtures/stylesheets/compass/sass
- `box-sizeing.scss` is misspelled
- some filenames are dash separated, some are underscore separated
- some files are written in SCSS syntax, some use Sass syntax

I propose that we decide upon a syntax and naming convention and rigidly enforce it. I suggest using dashes consistently and SCSS syntax.

**Comment by mirisuzanne** [comment]
Yes, you're right. Dashes and scss should be standard across the repo. I think that should be fixed in the master branch - otherwise it will lead to major conflicts when we merge.

**Comment by scottdavis** [comment]
This will take awhile. And will be low priority in my book until we get a new release out.
